website/content/maintainers/templates/governance-elections.md

```diff
@@ -8,6 +8,260 @@ aliases:
+assumes that you are using the [Contributor Ladder], since it is a good way to provide
+guidance about eligibility requirements. It is possible to implement this without a
+[Contributor Ladder], but it is not recommended.
+
+## Glossary
+
+**Project**: Your entire CNCF project, rather than individual repositories or subprojects.
+
+**Steering Committee**: The overall governing body for your project.
+
+## Template Details
+
+### Introduction
+
+Your intro paragraph needs to include a succinct summary of the Steering
+Committee and the scope of its oversight / decision-making.
+
+### Values
+
+Like the other templates, you need to place a list of values here that define
+what your project strives for.  Some of these will be general (like "fairness")
+and some will be specific to your problem domain (like "asyncronous operation").
+The Values are listed in your governance template because all project leaders
+are expected to follow these values.  Deciding your values is a good topic
+for a general community meeting.
+
+See our documentation on [Charters] for some examples.
+
+### Charter
+
+This section describes the responsibilities and powers belong to the Steering
+Committee, and should include a list of responsibilites and areas where the
+Steering Committee is responsible for oversight. Exactly what is in this section
+varies widely depending on your project structure. The template contains some
+examples of what you might expect to have in this section.
+
+### Delegated Authority
+
+List all other committees, groups, or other areas where the Steering Committee
+delegates responsibilities. This may include a security response committee, code
+of conduct committee, technical oversight committee, etc. For example, you may
+have a Marketing Committee that handles all work with CNCF marketing, or a
+different body that handles escalated technical disputes.
+
+### Committee Meetings
+
+You need to explicitly describe where, how, and when the Steering Committee will
+meet along with details about where the minutes will be published and how
+community members can contact the Steering Committee or raise issues for
+discussion.
+
+You should consider setting up a cadence of private vs. public meetings with the
+private meetings being for sensitive topics that should not be open to the
+public.
+
+### Committee Members
+
+When a steering committee governance model is first implemented, it usually
+starts with a bootstrap steering committee with members selected by the project
+based on who is currently making project decisions. This group will be
+responsible for running the project, but with a set deadline for how they will
+be replaced by an elected steering committee.
+
+**Question: do we have a good example of this that I can link to? I'm failing to
+find the original one from k8s**
+
+### Decision Process
+
+We recommend using consensus as the primary decision-making method, but you will
```

Comment:
We should be more explicit about how we get to consensus, and how we can help document decisions better.

Reply:
Good catch - I think I've addressed this in the most recent commit.